modules/kargoBidAdapter.js

```diff
@@ -98,7 +98,7 @@ function buildRequests(validBidRequests, bidderRequest) {
   });
 
   const reqCount = getRequestCount()
-  if (reqCount != null && reqCount > 0) {
+  if (reqCount != null) {
```

Comment:
0 represents the first request from the page, and BTO needs it: https://github.com/KargoGlobal/kraken/blob/master/handlers/bid.go#L361